apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -0,0 +1,230 @@
+                    id="cr-title"
+                    placeholder="Elementary Mathematics"
+                    minlength="1"
+                    required
+                  />
+                  <small class="form-text text-muted">
+                    This is the official title of the course, as given in the course catalog.
+                  </small>
+                </div>
+                <div class="form-group">
+                  <label for="cr-ghuser">GitHub Username (optional)</label>
+                  <input type="text" class="form-control" name="cr-ghuser" id="cr-ghuser" />
+                  <small class="form-text text-muted">
+                    Providing your GitHub username will allow you to edit course content offline.
+                    You do not need to provide this if you would like to use the online web editor.
+                  </small>
+                </div>
+                <div class="form-group">
+                  <label>Your Role in the Course</label>
+                  <div class="form-control">
+                    <input type="radio" id="role-instructor" name="cr-role" value="instructor" />
+                    <label for="role-instructor">Official Course Instructor</label>
+                  </div>
+                  <div class="form-control">
+                    <input type="radio" id="role-ta" name="cr-role" value="ta" />
+                    <label for="role-ta">Teaching Assistant or other course staff</label>
+                  </div>
+                  <div class="form-control">
+                    <input type="radio" id="role-admin" name="cr-role" value="admin" />
+                    <label for="role-admin">Institution Administrative Staff</label>
+                  </div>
+                  <div class="form-control">
+                    <input type="radio" id="role-student" name="cr-role" value="student" />
+                    <label for="role-student">Student</label>
+                  </div>
+                  <div
+                    style="display: none;"
+                    class="role-comment role-comment-ta role-comment-admin form-text card"
+                  >
+                    <p class="card-body">
+                      <b>A new course instance must be requested by the instructor.</b> Please ask
+                      the official course instructor to submit this form.
+                    </p>
+                  </div>
+                  <div
+                    style="display: none;"
+                    class="role-comment role-comment-student form-text card"
+                  >
+                    <p class="card-body">
+                      <b>This is the wrong form for you.</b> If you would like to enroll in an
+                      existing course, please use the
+                      <a href="enroll">form to Enroll in a course</a>. If your course is not listed
+                      there, your instructor is not yet using PrairieLearn for the course.
```

Comment:
I know this is the text that already existed before, but I wonder if this should be changed to phase out the enroll form, or at the very least point the student to the instructor, since they may be hiding the course and using a link in the LMS/course page.

Reply:
Yeah, you're right that changing prematurely may not be a good idea, but I'd prefer some adjustments at least to the last sentence. Something like:

> If your course is not listed there, contact your instructor for instructions on how to access your assessments.

Reply:
Changed in c1a0a26a